internal/provider/machine_pool_resource.go

```diff
@@ -77,6 +78,10 @@ func machinePoolResourceAttributes() map[string]schema.Attribute {
 			MarkdownDescription: "Name of the primary disk of the machine, i.e. /dev/vda",
 			Required:            true,
```

Comment:
@boebu I think this should now only be required if `reuse_existing_root_partition` is `false`.